<?xml version="1.0"?>
<launch>

    <node pkg="youBot_placing_experiment" type="youBot_placing_node"
            name="youBot_placing" output="screen">

        <!-- the cycle rate of the node (in Hz) -->
        <param name="loop_rate" type="double" value="10"/>

        <!-- the name of robot arm used in planning -->
        <param name="arm" type="string" value="arm_1"/>

        <!-- the name of robot gripper used in planning -->
        <param name="gripper" type="string" value="arm_1_gripper"/>

        <!-- start and end robot configuration used in planning -->
        <param name="start_state" type="string" value="folded"/>

        <param name="yb_number" type="int" value="3"/>

    </node>
    <include file="$(find mcr_rosbag_recorder)/ros/launch/rosbag_recorder.launch"/>
    <node pkg="mcr_rosbag_recorder" type="read_tf.py" name="read_tf" output="screen"/>
    <node pkg="tf" type="static_transform_publisher" name="static_tf_pub" args="0.161 0.711 0 0 3.141 0 arm_link_0 world_frame 10" output="screen"/>
</launch>
